Source/AssistantV1/Assistant.swift

```diff
@@ -102,6 +102,11 @@ public class Assistant {
             return
         }
 
+        // construct header parameters
+        var headers = defaultHeaders
+        headers["Accept"] = "application/json"
+        headers["Content-Type"] = "application/json"
+
```

Comment:
It looks like all the changes in the service file are from template changes unrelated to the new version of assistant.  I suppose that's fine ... it just looks a little odd.